File: android/app/src/main/java/jagtheora/ogg/OggPacket.java
package jagtheora.ogg;

import jagtheora.misc.SimplePeer;

public final class OggPacket extends SimplePeer {

    public boolean isTheora() { return false; }

    public int isHeader() { return 0; }

    protected void clear() {}

    public byte[] getData() { return null; }

    public int isKeyFrame() { return 0; }

    public boolean isVorbis() { return false; }
}

File: src/jvmMain/kotlin/jagtheora/ogg/OggPacket.kt
package jagtheora.ogg

import jagtheora.misc.SimplePeer

actual class OggPacket : SimplePeer() {
    actual val isTheora: Boolean
        external get

    actual external override fun clear()

    actual val data: ByteArray?
        external get

    /*private*/ actual val isHeader: Int
        external get

    actual val isKeyFrame: Int
        external get

    actual val isVorbis: Boolean
        external get
}
